app/assets/stylesheets/active_admin/_header.scss

```diff
@@ -145,11 +123,8 @@
     padding-right: 26px;
     text-align: right;
 
-    a { text-decoration: none; }
-    a:hover { color: #fff; }
-
-    li {
-      display:inline;
+    a:hover {
+      color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.